src/TextField/TextField.js

```diff
@@ -356,6 +361,14 @@ class TextField extends Component {
     }
   };
 
+  handleComposition = (event) => {
```

Comment:
Please could you add a TODO comment explaining why this is needed (reference the FB issue), so we can revisit and remove the workaround if / when Facebook finally fix it (there is a PR pending review since Dec 2016).